### PR TITLE
Add built-in monitoring

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-tidb-cluster.md [hunk 1]
```diff
@@ -45,13 +45,32 @@ You can see the current status of each running cluster on the cluster page.
...
+In TiDB Cloud, you can view the commonly used metrics of a cluster from the following pages:
+
+- Cluster overview page
+- Cluster monitoring page
+
+### Metrics on the cluster overview page
+
+The cluster overview page provides general metrics of a cluster, including Total QPS, Latency, Connections, TiFlash Request QPS, TiFlash Request Duration, TiFlash Storage Size, TiKV Storage Size, TiDB CPU, TiKV CPU, TiKV IO Read, and TiKV IO Write.
+
+To open the cluster overview page, take the following steps:
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:57]:
```suggestion
To view metrics on the cluster overview page, take the following steps:
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 2]
```diff
@@ -45,13 +45,32 @@ You can see the current status of each running cluster on the cluster page.
...
 1. Navigate to the **Active Clusters** page.
 
-2. Click the name of a cluster to go to the cluster overview page.
+2. Click the name of a cluster to go to its cluster overview page.
+
+### Metrics on the cluster monitoring page
+
+The cluster monitoring page provides a full standard metrics of a cluster. By viewing these metrics, you can easily identify performance issues and determine whether your current database deployment meets your requirements.
+
+To open the cluster monitoring page, take the following steps:
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:67]:
```suggestion
To view metrics on the cluster monitoring page, take the following steps:
```

File: TOC-tidb-cloud.md [hunk 3]
```diff
@@ -46,6 +46,7 @@
   - [Restore](/tidb-cloud/backup-and-restore.md#restore)
 - Monitor and Alert
   - [Overview](/tidb-cloud/monitor-tidb-cluster.md)
+  - [Built-in Monitoring](tidb-cloud/monitor-built-in-monitoring)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:49]:
```suggestion
  - [Built-in Monitoring](/tidb-cloud/monitor-built-in-monitoring)
```

**Comment by lilyjazz** [comment]
LGTM

File: tidb-cloud/monitor-built-in-monitoring.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+| TiFlash IO MBps | instances-write, instances-read | The total bytes of read and write in each TiFlash instance. |
+| TiFlash Storage Usage | instances | The storage size per TiFlash instance. |
+
+## FAQ
+
+### 1. Why are some panels empty on this page？
+
+If there is no value in the panel, there may be several scenarios.
+- The cluster's workload does not trigger this metric. For example, the failed query metric is always empty in the case of no failed queries.
+- The cluster version is low, please upgrade to the latest version of TiDB to see these metrics.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-monitoring.md:109]:
```suggestion
- The cluster version is low. You need to upgrade it to the latest version of TiDB to see these metrics.
```

File: tidb-cloud/monitor-built-in-monitoring.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+- If all the above scenarios are excluded, please contact technical support.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-monitoring.md:110]:
```suggestion

If all these reasons are excluded, you can contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) for troubleshooting.
```

File: tidb-cloud/monitor-built-in-monitoring.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+>
+> For [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier), the Monitoring page is unavailable. You can upgrade to [Dedicated Tier clusters](/tidb-cloud/select-cluster-tier.md#dedicated-tier) to use it.
+
+## How to use feature 
+
+To view these metrics, take the following steps:
+
+1. Navigate to the **Diagnostics** tab of a cluster.
+
+2. Click the **Monitoring** tab.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-monitoring.md:20]:
```suggestion
1. In the TiDB Cloud console, navigate to the **Active Clusters** page for your project, and then click the name of a cluster to go to its overview page.
2. Click the **Diagnostics** tab.
3. Click the **Monitoring** tab.
```

File: tidb-cloud/monitor-built-in-monitoring.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+| TiDB CPU Usage | instances | The statistics of CPU usage of each TiDB instance. |
+| TiDB Memory Usage | instances | The memory usage statistics of each TiDB instance. |
+| TiKV Uptime | instances | The runtime of each TiKV instance since last restart. |
+| TiKV CPU Usage | instances | The statistics of CPU usage of each TiKV instance. |
+| TiKV Memory Usage | instances | The memory usage statistics of each TiKV instance. |
+| TiKV IO MBps | instances-write, instances-read | The total bytes of read and write in each TiKV instance. |
+| TiKV Storage Usage | instances | The storage size per TiKV instance. |
+| TiFlash Uptime | instances | The runtime of each TiFlash instance since last restart. |
+| TiFlash CPU Usage | instances | The statistics of CPU usage of each TiFlash instance. |
+| TiFlash Memory Usage | instances | The memory usage statistics of each TiFlash instance. |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-monitoring.md:99]:
```suggestion
| TiFlash Memory  | instances | The memory usage statistics of each TiFlash instance. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 8]
```diff
@@ -45,13 +45,32 @@ You can see the current status of each running cluster on the cluster page.
...
+To view metrics on the cluster overview page, take the following steps:
 
 1. Navigate to the **Active Clusters** page.
 
-2. Click the name of a cluster to go to the cluster overview page.
+2. Click the name of a cluster to go to its cluster overview page.
+
+### Metrics on the cluster monitoring page
+
+The cluster monitoring page provides a full standard metrics of a cluster. By viewing these metrics, you can easily identify performance issues and determine whether your current database deployment meets your requirements.
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:65]:
```suggestion
The cluster monitoring page provides a full set of standard metrics of a cluster. By viewing these metrics, you can easily identify performance issues and determine whether your current database deployment meets your requirements.
```

File: tidb-cloud/tune-performance.md [hunk 9]
```diff
@@ -15,11 +15,13 @@ TiDB Cloud provides [Statement Analysis](#statement-analysis) and [Key Visualize
 
 To use the statement analysis, perform the following steps:
 
-1. Navigate to the **Diagnostics** tab of a cluster. The **Statement** sub-tab displays by default.
+1. Navigate to the **Diagnostics** tab of a cluster.
+
+2. Click the **Statement** sub-tab.
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:20]:
```suggestion
2. Click the **Statement** tab.
```

File: tidb-cloud/tune-performance.md [hunk 10]
```diff
@@ -33,7 +35,7 @@ To view the key analytics, perform the following steps:
 
 1. Navigate to the **Diagnostics** tab of a cluster.
 
-2. Select the **Key Visualizer** tab.
+2. Click the **Key Visualizer** sub-tab.
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:38]:
```suggestion
2. Click the **Key Visualizer** tab.
```

File: tidb-cloud/select-cluster-tier.md [hunk 11]
```diff
@@ -77,7 +77,8 @@ Alternatively, you can log in to the TiDB Cloud console, and then click **Resume
 - You cannot create any changefeeds (Apache Kafka Sink and MySQL Sink) or use [TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview) to replicate incremental data.
 - You cannot use VPC Peering to connect to clusters.
 - You cannot scale clusters to larger storage, standard nodes, or increase the number of nodes.
-- You cannot use a third-party monitoring service.
+- You cannot entry the [Monitoring page](/tidb-cloud/monitor-built-in-monitoring.md) to use the standard monitoring service.
```
Comment (qiancai) [inline tidb-cloud/select-cluster-tier.md:80]:
```suggestion
- You cannot view the [Monitoring page](/tidb-cloud/monitor-built-in-monitoring.md).
```

File: tidb-cloud/tune-performance.md [hunk 12]
```diff
@@ -15,11 +15,13 @@ TiDB Cloud provides [Statement Analysis](#statement-analysis) and [Key Visualize
 
 To use the statement analysis, perform the following steps:
 
-1. Navigate to the **Diagnostics** tab of a cluster. The **Statement** sub-tab displays by default.
+1. Navigate to the **Diagnostics** tab of a cluster.
```
Comment (hfxsd) [inline tidb-cloud/tune-performance.md:18]:
```suggestion
1. Navigate to the **Diagnosis** tab of a cluster.
```

File: tidb-cloud/tune-performance.md [hunk 13]
```diff
@@ -33,7 +35,7 @@ To view the key analytics, perform the following steps:
 
 1. Navigate to the **Diagnostics** tab of a cluster.
```
Comment (hfxsd) [inline tidb-cloud/tune-performance.md:36]:
```suggestion
1. Navigate to the **Diagnosis** tab of a cluster.
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 14]
```diff
@@ -45,13 +45,32 @@ You can see the current status of each running cluster on the cluster page.
...
+
+To view metrics on the cluster monitoring page, take the following steps:
+
+1. Navigate to the **Diagnostics** tab of a cluster.
```
Comment (hfxsd) [inline tidb-cloud/monitor-tidb-cluster.md:69]:
```suggestion
1. Navigate to the **Diagnosis** tab of a cluster.
```

File: tidb-cloud/monitor-built-in-monitoring.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+
+> **Note:**
+>
+> For [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier), the Monitoring page is unavailable. You can upgrade to [Dedicated Tier clusters](/tidb-cloud/select-cluster-tier.md#dedicated-tier) to use it.
+
+## How to use feature 
+
+To view these metrics, take the following steps:
+
+1. Navigate to the **Diagnostics** tab of a cluster.
```
Comment (hfxsd) [inline tidb-cloud/monitor-built-in-monitoring.md:18]:
```suggestion
1. Navigate to the **Diagnosis** tab of a cluster.
```

File: tidb-cloud/monitor-built-in-monitoring.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+| Database Time by SQL Phase | get token, parse, compile, execute | Database time consumed in four SQL processing phases: get token, parse, compile, and execute. The SQL execution phase is in green and other phases are in red on general. If non-green areas take a large proportion, it means most database time is consumed by other phases than the execution phase and further cause analysis is required. |
+| SQL Execute Time Overview | tso_wait, Get, Cop, Commit, etc. | Green metrics stand for common KV write requests (such as prewrite and commit), blue metrics stand for common read requests, and metrics in other colors stand for unexpected situations which you need to pay attention to. For example, pessimistic lock KV requests are marked red and TSO waiting is marked dark brown.If non-blue or non-green areas take a large proportion, it means there is a bottleneck during SQL execution. For example, if serious lock conflicts occur, the red area will take a large proportion. If excessive time is consumed in waiting TSO, the dark brown area will take a large proportion. |
```
Comment (hfxsd) [inline tidb-cloud/monitor-built-in-monitoring.md:35]:
```suggestion
| SQL Execute Time Overview | tso_wait, Get, Cop, Commit, etc. | Green metrics stand for common KV write requests (such as prewrite and commit), blue metrics stand for common read requests, and metrics in other colors stand for unexpected situations which you need to pay attention to. For example, pessimistic lock KV requests are marked red and TSO waiting is marked dark brown. If non-blue or non-green areas take a large proportion, it means there is a bottleneck during SQL execution. For example, if serious lock conflicts occur, the red area will take a large proportion. If excessive time is consumed in waiting TSO, the dark brown area will take a large proportion. |
```

File: TOC-tidb-cloud.md [hunk 17]
```diff
@@ -46,6 +46,7 @@
   - [Restore](/tidb-cloud/backup-and-restore.md#restore)
 - Monitor and Alert
   - [Overview](/tidb-cloud/monitor-tidb-cluster.md)
+  - [Built-in Monitoring](/tidb-cloud/monitor-built-in-monitoring.md)
```
Comment (lilin90) [inline TOC-tidb-cloud.md:49]: Suggest removing the redundant "monitor".
```suggestion
  - [Built-in Monitoring](/tidb-cloud/built-in-monitoring.md)
```

File: tidb-cloud/built-in-monitoring.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+| Disconnection | Instances | The number of clients disconnected to each TiDB instance. |
+
+### SQL Count
+
+| Metric name  | Labels | Description                                   |
+| :------------| :------| :-------------------------------------------- |
+| Query Per Second | Select, Insert, Use, etc. | The number of SQL statements executed per second in all TiDB instances, which are collected by types, such as `SELECT`, `INSERT`, and `UPDATE`. |
+| Failed Queries | Error types | The statistics of error types (such as syntax errors and primary key conflicts) according to the SQL statement execution errors per minute on each TiDB instance. It contains the module in which an error occurs and the error code. |
+| Command Per Second | Query, StmtExecute, StmtPrepare, etc. | The number of commands processed by all TiDB instances per second based on command types. |
+| Queries Using Plan Cache OPS | hit, miss | The number of queries using plan cache per second in all TiDB instances. |
```
Comment (Yui-Song) [inline tidb-cloud/built-in-monitoring.md:51]: 还有miss的是不是应该也描述一下

**Review by Yui-Song — APPROVED** [review]

File: tidb-cloud/built-in-monitoring.md (hunk 18 above)
Comment (lilyjazz) [inline tidb-cloud/built-in-monitoring.md:51]:
```suggestion
| Queries Using Plan Cache OPS | hit, miss | The number of queries using or missing plan cache per second in all TiDB instances. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+| Disconnection | Instances | The number of clients disconnected to each TiDB instance. |
+
+### SQL Count
+
+| Metric name  | Labels | Description                                   |
+| :------------| :------| :-------------------------------------------- |
+| Query Per Second | Select, Insert, Use, etc. | The number of SQL statements executed per second in all TiDB instances, which are collected by types, such as `SELECT`, `INSERT`, and `UPDATE`. |
+| Failed Queries | Error types | The statistics of error types (such as syntax errors and primary key conflicts) according to the SQL statement execution errors per minute on each TiDB instance. It contains the module in which an error occurs and the error code. |
+| Command Per Second | Query, StmtExecute, StmtPrepare, etc. | The number of commands processed by all TiDB instances per second based on command types. |
+| Queries Using Plan Cache OPS | hit, miss | The number of queries using or missing plan cache per second in all TiDB instances. |
```
Comment (lilyjazz) [inline tidb-cloud/built-in-monitoring.md:51]:
```suggestion
| Queries Using Plan Cache OPS | hit, miss | Hit: the number of queries using plan cache per second in all TiDB instances. Miss: the number of queries missing plan cache per second in all TiDB instances. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 20]
```diff
@@ -0,0 +1,111 @@
...
+| Disconnection | Instances | The number of clients disconnected to each TiDB instance. |
+
+### SQL Count
+
+| Metric name  | Labels | Description                                   |
+| :------------| :------| :-------------------------------------------- |
+| Query Per Second | Select, Insert, Use, etc. | The number of SQL statements executed per second in all TiDB instances, which are collected by types, such as `SELECT`, `INSERT`, and `UPDATE`. |
+| Failed Queries | Error types | The statistics of error types (such as syntax errors and primary key conflicts) according to the SQL statement execution errors per minute on each TiDB instance. It contains the module in which an error occurs and the error code. |
+| Command Per Second | Query, StmtExecute, StmtPrepare, etc. | The number of commands processed by all TiDB instances per second based on command types. |
+| Queries Using Plan Cache OPS | hit, miss | Hit: the number of queries using plan cache per second in all TiDB instances. Miss: the number of queries missing plan cache per second in all TiDB instances. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:51]:
```suggestion
| Queries Using Plan Cache OPS | hit, miss | Hit: the number of queries using plan cache per second in all TiDB instances. <br/> Miss: the number of queries missing plan cache per second in all TiDB instances. |
```

**Review by hfxsd — APPROVED** [review]
LGTM

File: tidb-cloud/monitor-tidb-cluster.md [hunk 21]
```diff
@@ -45,13 +45,32 @@ You can see the current status of each running cluster on the cluster page.
...
 
 1. Navigate to the **Active Clusters** page.
 
-2. Click the name of a cluster to go to the cluster overview page.
+2. Click the name of a cluster to go to its cluster overview page.
+
+### Metrics on the cluster monitoring page
+
+The cluster monitoring page provides a full set of standard metrics of a cluster. By viewing these metrics, you can easily identify performance issues and determine whether your current database deployment meets your requirements.
+
```
Comment (qiancai) [inline tidb-cloud/monitor-tidb-cluster.md:66]:
```suggestion

> **Note:**
>
> Currently, the Monitoring page is unavailable for [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier).

```

File: tidb-cloud/built-in-monitoring.md [hunk 22]
```diff
@@ -0,0 +1,111 @@
...
+| TiFlash IO MBps | instances-write, instances-read | The total bytes of read and write in each TiFlash instance. |
+| TiFlash Storage Usage | instances | The storage size per TiFlash instance. |
+
+## FAQ
+
+**1. Why are some panes empty on this page?**
+
+If a pane does not provide any metrics, the possible reasons are as follows:
+- The workload of the corresponding cluster does not trigger this metric. For example, the failed query metric is always empty in the case of no failed queries.
+- The cluster version is low. You need to upgrade it to the latest version of TiDB to see these metrics.
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:109]:
```suggestion
If a pane does not provide any metrics, the possible reasons are as follows:

- The workload of the corresponding cluster does not trigger this metric. For example, the failed query metric is always empty in the case of no failed queries.
- The cluster version is low. You need to upgrade it to the latest version of TiDB to see these metrics.
```

**Pull request merged**